README.md

```diff
@@ -415,6 +361,139 @@ be lifted in the future.
+
+By default the tests will be run against `enterprise-master` and the latest released version. If you want to test on a custom branch you should change the config files of relevant tests with your custom branch name in:
+
+```text
+postgres_citus_versions: [('12.1', 'your-custom-branch-name-in-enterprise'), ('12.1', 'release-9.1')]
+```
+
+You can change all the settings in these files, the config files for tests are located at:
+
+* pgbench: https://github.com/citusdata/test-automation/tree/master/fabfile/pgbench_confs
+* scale: https://github.com/citusdata/test-automation/tree/master/fabfile/pgbench_confs
+* tpch: https://github.com/citusdata/test-automation/tree/master/fabfile/tpch_confs
+
+By default, the following tests will be run for each test:
+
+* pgbench: `pgbench_default.ini` and `pgbench_default_without_transaction.ini`
+* scale: `scale_test.ini`
+* tpch: `tpch_default.ini`
+
+If you dont want to use default cluster settings(instance types etc), you can change them in https://github.com/citusdata/test-automation/blob/master/azure/azuredeploy.parameters.json.
+
+If you want to change how long each test will be run, you can change the times with the `-T` parameter. https://github.com/citusdata/test-automation/blob/master/fabfile/pgbench_confs/pgbench_default.ini#L33 
+
+```
+pgbench_command: pgbench -c 32 -j 16 -T <test time in seconds> -P 10 -r
+```
+
+## <a name="running-automated-hammerdb"></a>Running Automated Hammerdb
+
+Hammerdb tests are run from a driver node. Driver node is in the same virtual network as the cluster.
+You can customize the hammerdb cluster in the `hammerdb` folder using `azuredeploy.parameters.json`.
+
+**You should create a new branch and change the settings in the new branch and push the branch so that
+when the tool clones the repository it can download your branch.**
+
+In order to run hammerdb benchmark:
+
+```bash
+eval `ssh-agent -s`
+ssh-add
+export RESOURCE_GROUP_NAME=<your resource group name>
+export GIT_USERNAME=<Your github username>
+export GIT_TOKEN=<Your github token with at least write and read access> # You can create a github token from https://github.com/settings/tokens.
+cd hammerdb
+# YOU SHOULD CREATE A NEW BRANCH AND CHANGE THE SETTINGS IN THE NEW BRANCH
+# AND PUSH THE BRANCH SO THAT WHEN THE TOOL CLONES THE REPOSITORY
+# IT CAN DOWNLOAD YOUR BRANCH.
+./create-run.sh
+# you will be given a command to connect to the driver node and what
+# to run afterwards.
+```
+
+**After running ./create-run.sh you do not have to be connected to the driver node at all, it will take care of the rest for you.**
+
+**You need to delete the cluster yourself(You can do that with azure/delete-resource-group.sh or from the portal). In hammerdb, we sometimes need to connect to the coordinator or driver node to get some more information.**
```

Comment:
Very minor; `You need to` -> `You are responsible for deleting the cluster / resource group yourself`.


> In hammerdb, we sometimes need to connect to the coordinator or driver node to get some more information.

I'm ok to remove this sentence as it might be confusing.